pkg/daemon/update.go

```diff
@@ -997,16 +1020,32 @@ func origParentDir() string {
 	return filepath.Join("/etc", "machine-config-daemon", "orig")
 }
 
+func noOrigParentDir() string {
+	return filepath.Join("/etc", "machine-config-daemon", "noorig")
+}
+
 func origFileName(fpath string) string {
 	return filepath.Join(origParentDir(), fpath+".mcdorig")
 }
 
+func noOrigFileName(fpath string) string {
```

Comment:
i dont really understand this naming?  Can we make it clearer since it's not really clear what's going on here... and the naming of this noOrig doesn't really make it clear

Reply:
ie anyone who will have to maintain this in the future is going to have trouble understanding what this is for...

Reply:
I can add some comments above this as well. To be clear this is to unblock upgrades for 4.4 and not the most clean long term solution. We are attempting to fix an insidious error that we didn't catch way back